tests/integration/mutation/update/field_kinds/one_to_many/with_alias_test.go

```diff
@@ -247,7 +233,23 @@ func TestMutationUpdateOneToMany_InvalidAliasRelationNameToLinkFromManySide_Coll
 					}`,
 					invalidAuthorKey,
 				),
-				ExpectedError: "The given field does not exist. Name: author",
+				// ExpectedError: "The given field does not exist. Name: author",
```

Comment:
todo: Please remove the commented out error.

praise: Thank you for fixing this!

Reply:
✅